docs/backends/bind.rst

```diff
@@ -144,6 +144,15 @@ will be loaded at first request.
 .. note::
   This does not add the zone to the :ref:`setting-bind-config` file.
 
+``bind-domain-extended-status <domain> [domain]``
```

Comment:
Looks like you wrote this one before you made the arguments entirely optional.

Reply:
Fixed, thanks!